iceoryx_posh/test/moduletests/test_popo_unique_port_id.cpp

```diff
@@ -26,160 +28,155 @@ using namespace ::testing;
 using namespace iox::popo;
 using namespace iox::cxx;
 
-TEST(TypedUniqueId_RouDiId, SettingTheRouDiIdWorks)
+TEST(UniquePortId_test, SettingTheRouDiIdWorks)
 {
     ::testing::Test::RecordProperty("TEST_ID", "473467bf-1a6f-4cd2-acd8-447a623a5301");
     uint16_t someId = 1243u;
```

Comment:
Do `U` fancy capital literals? :smirk:

Reply:
Well, I find them horrible https://github.com/eclipse-iceoryx/iceoryx/discussions/617 ... but it seems almost nobody agrees and I have to live this nightmare day by day 😜